frontend/src/main/scala/Frontend.scala

```diff
@@ -84,7 +84,7 @@ class NiFrontend(json: String) extends js.Object {
    */                                                                
   object weeklyContributions extends js.Object {
 
-    def apply(
+    def calculate(
       from: LocalDate,
```

Comment:
`Date` works better for JS native dates being passed as params - not sure what we need to do to get `LocalDate` working...